Specs/Data/Models/GltfLoader/BoxUnlit/glTF/BoxUnlit.gltf

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
This model exists in `Models/PBR/BoxUnlit`, could you remove this file and change the spec to link to that one?